packages/ckb/src/utils/ckb-tx.ts

```diff
@@ -139,3 +141,27 @@ export const isScriptEqual = (s1: Hex | CKBComponents.Script, s2: Hex | CKBCompo
   const temp2 = typeof s2 === 'string' ? remove0x(s2) : remove0x(serializeScript(s2));
   return temp1 === temp2;
 };
+
+/**
+ * Check whether the capacity of inputs is sufficient for outputs
+ * @param ckbTx CKB raw transaction
+ * @param collector The collector that collects CKB live cells and transactions
+ * @returns When the capacity of inputs is sufficient for outputs, return true, otherwise return false
+ */
+export const checkCkbTxInputsCapacitySufficient = async (
+  ckbTx: CKBComponents.RawTransaction,
+  collector: Collector,
+): Promise<boolean> => {
+  let sumInputsCapacity = BigInt(0);
+  for await (const input of ckbTx.inputs) {
+    const liveCell = await collector.getLiveCell(input.previousOutput!);
+    if (!liveCell) {
+      throw new NoLiveCellError('The cell with the specific out point is dead');
```

Comment:
Is it the same if returning `false` when the cell is not lived? 
Didn't expect that the user has to wrap a try catch scope by looking at the signature.

Reply:
After discussion, we think throwing an error here can help users identify internal errors. 
The btc lib doesn't catch this function, and just leaves it to the users.